psst-gui/src/ui/album.rs

```diff
@@ -99,14 +98,15 @@ fn rounded_cover_widget(size: f64) -> impl Widget<Arc<Album>> {
     cover_widget(size).clip(Size::new(size, size).to_rounded_rect(4.0))
 }
 
-pub fn album_widget() -> impl Widget<WithCtx<Arc<Album>>> {
-    let album_cover = rounded_cover_widget(theme::grid(6.0));
-
-    let album_name = Flex::row()
+pub fn album_widget(horizontal: bool) -> impl Widget<WithCtx<Arc<Album>>> {
+    let (album_cover_size, album_name_layout) = if horizontal { (16.0, Flex::column()) } else { (6.0, Flex::row()) };
+    let album_cover = rounded_cover_widget(theme::grid(album_cover_size));
+    
+    let album_name = album_name_layout
         .with_child(
             Label::raw()
                 .with_font(theme::UI_FONT_MEDIUM)
-                .with_line_break_mode(LineBreaking::Clip)
+                .with_line_break_mode(LineBreaking::WordWrap)
```

Comment:
Is there a reason you switched to WordWrap for just artists? I do worry that there might be a super long artist name that will cause it to look ugly :)